README.md

```diff
@@ -198,49 +197,25 @@ In a new terminal window run: `redis-cli`
 
 Now type `ping` and it should respond `pong`
 
-### Sass
-
-`gem install sass`
-
-If you get a permissions error, DONT SUDO!
-
-Instead change ownership of that directory
-
-`sudo chown -R $(whoami) /Library/Ruby/Gems/2.0.0` (or whatever ruby version you have)
-
-Then run the command again
-
-## Dot files
-
-My vim config can be found [here](https://github.com/shouston3/My-Config/blob/master/.vimrc) and dowloaded with:
-
-`curl https://cdn.rawgit.com/shouston3/My-Config/master/.vimrc > ~/.vimrc`
-
-My shell preference is the zsh shell with the oh-my-zsh extensions, see why [here](https://github.com/shouston3/My-Config/blob/master/WHY_ZSH.md)
-
-My terminal theme can be found [here](https://github.com/shouston3/My-Config/blob/master/sams-theme.zsh-theme) and after you have oh-my-zsh installed, install with:
-
-`curl https://cdn.rawgit.com/shouston3/My-Config/master/sams-theme.zsh-theme > ~/.oh-my-zsh/themes`
-
-Then set up by changing `ZSH_THEME="robbyrussel"` to `ZSH_THEME="sams-theme"`
+----
 
-Your terminal should look something like this:
+#### Heroku toolbelt
 
-<img src="imgs/terminal.png" />
 
-[Here](https://github.com/shouston3/My-Config/blob/master/sams-aliases.plugin.zsh) are the functions and aliases I use day to day
+#### Misc
 
-They can be installed with:
+<img src="http://core0.staticworld.net/images/article/2013/10/system-preferences-gallery-100065979-large.png" align="right" height="150px" />
```

Comment:
@shouston3 tip: if you copy the image an paste it in an issue/PR comment:
![image](https://user-images.githubusercontent.com/194400/27246623-a8dbf7f8-52ea-11e7-9581-e02c9ab09241.png)
GitHub will "upload" the image to their CDN (S3) which means there's no risk of it vanishing. :-)